<launch>
	<!-- ========================================================== -->
	<!-- Launch everything needed for Kinefly, except the           -->
   	<!-- source image streams.                                      -->
	<!-- ========================================================== -->	

	<include file="$(find Kinefly)/launch/$(env RIG)/params_kinefly.launch"   	ns="kinefly" 	/>
	<!-- include file="$(find Kinefly)/launch/$(env RIG)/params_phidgetsanalog.launch" 		/ -->

	<node name="kinefly"     	pkg="Kinefly"   	type="kinefly.py"      	required="true" />

	<!-- node name="ReadAI"     	pkg="daq_test_1"   	type="ReadAI.py"      	required="false" / -->

	<!-- node name="flystate2phidgetsanalog"    pkg="Kinefly"   type="flystate2phidgetsanalog_new.py" ns="kinefly -->"  required="true"	/>

</launch>
